### ISSUE TITLE
:bug: Arch missmatch for opensuse leap v2.4.1

### ISSUE THREAD

**Issue by mauromorales** [kairos-io/kairos#1981]
```
$ uname -a
Linux zeno 6.5.0-10-generic #10-Ubuntu SMP PREEMPT_DYNAMIC Fri Oct 13 13:49:38 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
$ docker run --rm -ti quay.io/kairos/core-opensuse-leap:v2.4.1
Unable to find image 'quay.io/kairos/core-opensuse-leap:v2.4.1' locally
v2.4.1: Pulling from kairos/core-opensuse-leap
a8dc0ee650cf: Already exists 
8def229cd835: Pull complete 
e6abe94e4eca: Pull complete 
2f4fc1057378: Pull complete 
2e0c7faebd15: Pull complete 
4f4fb700ef54: Pull complete 
4eccb1c183eb: Pull complete 
d6e0fad6c837: Pull complete 
9be66e9d4765: Pull complete 
7db608aa4637: Pull complete 
9567db38d2d4: Pull complete 
866e80044ad0: Pull complete 
925b781fd521: Pull complete 
78a876a64a4a: Pull complete 
e4d16a8eda23: Pull complete 
61673fecbdf2: Pull complete 
cb8effb0635f: Pull complete 
a9009484db86: Pull complete 
Digest: sha256:b32c51ecda17d29551dd23e0ea36a0064a8b135b8889868aa8466c3265dcec11
Status: Downloaded newer image for quay.io/kairos/core-opensuse-leap:v2.4.1
WARNING: The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64/v3) and no specific platform was requested

```

**Comment by jimmykarily** [comment]
True... https://quay.io/repository/kairos/core-opensuse-leap/manifest/sha256:b32c51ecda17d29551dd23e0ea36a0064a8b135b8889868aa8466c3265dcec11

how could that be?

**Comment by jimmykarily** [comment]
I think the all-arm target used the exact same name, thus it overwrote what the other target has pushed.

**Comment by jimmykarily** [comment]
Same issue on 2.4.0:

```
Status: Downloaded newer image for quay.io/kairos/core-opensuse-leap:v2.4.0
WARNING: The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64/v3) and no specific platform was requested

```

**Comment by jimmykarily** [comment]
Added a warning on 2.4.0 and 2.4.1. We will fix it on 2.4.2
